include/nn/ccr/cdc_sys.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <wut.h>
+#include <coreinit/time.h>
+
+/**
+ * \defgroup nn_ccr_cdc_sys
+ * \ingroup nn_ccr
+ * (see nn::ccr)
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum CCRCDCSysDrhStateEnum
+{
+    CCR_CDC_SYS_DRH_STATE_ECO  = 0x04,
+    CCR_CDC_SYS_DRH_STATE_CAFE = 0xFF,
+} CCRCDCSysDrhStateEnum;
+
+typedef struct CCRCDCSysDrhState {
```

Comment:
Hmm for `CCRCDCSysSetDrcState` I simply made this a `uint8_t` typedef, but if they pass a pointer to the set functions it might have been a struct originally?

Reply:
For me it looks like this. Should I change `CCRCDCSysSetDrcState` and friends?

Reply:
Yeah, that works.